lib/hashie/mash.rb

```diff
@@ -62,59 +62,12 @@ module Hashie
   class Mash < Hash
     include Hashie::Extensions::PrettyInspect
     include Hashie::Extensions::RubyVersionCheck
+    extend Hashie::Extensions::KeyConflictWarning
+    const_set :CannotDisableMashWarnings,
```

Comment:
I feel like this is unnecessary, just use `Hashie::Extensions::KeyConflictWarning::CannotDisableMashWarnings`?

Reply:
True, I was thinking about backwards compatibility, do you think anyone relies on rescuing that error? I'd err on not

Reply:
I highly doubt it, but we could just note that in `UPGRADING.md` in case anyone does.

Reply:
👍